cypress/e2e/editor.cy.ts

```diff
@@ -82,8 +111,27 @@ function embedEdusharingAsset() {
   cy.wait(6000)
 }
 
+function changeContent() {
+  // Create a new plugin
+  cy.get('div.add-trigger').eq(1).click()
+  cy.contains('Box').click()
```

Comment:
This searches for an HTML element containing "Box" in the whole dom. If this string can be found elsewhere, this click might fail. I had this issue once. As a quick fix, we can search for "Edusharing Inhalte" instead, since this is less likely to appear elsewhere. Sadly, targeting the correct think to click using css classes is difficult.

Reply:
Implemented!